src/index.js

```diff
@@ -106,87 +91,80 @@ function initializePal(loader) {
-    .then(palModule => type === 'nodejs' && !isInitialized && palModule.globalize() || palModule.initialize());
+               .then(palModule => type === 'nodejs' && !isInitialized && palModule.globalize() || palModule.initialize());
 }
 
 function preparePlatform(loader) {
-  // Note: Please do NOT add PLATFORM.moduleName() others than 'aurelia-framework'.
-  //       This is the _only_ module actually loaded by the bootstrapper.
-  //       The other ones are fake dependencies, if you look at the code carefully you'll 
-  //       notice that they are just some name resolutions and mapping; nothing gets loaded.
 
+  const map = (moduleId, relativeTo) => 
+    loader.normalize(moduleId, relativeTo)
+          .then(normalized => {
+            loader.map(moduleId, normalized);
+            return normalized;
+          });
+  
   return initializePal(loader)
     .then(() => loader.normalize('aurelia-bootstrapper'))
     .then(bootstrapperName => {
       // aurelia-framework re-exports pretty much everything.
       // As can be seen at the end of this method, the only field accessed by bootstrapper is `Aurelia`,
       // so we document that to enable tree shaking on all other exported members.
-      return loader.normalize(PLATFORM.moduleName('aurelia-framework', { exports: ['Aurelia'] }),
-                              bootstrapperName)
-        .then(frameworkName => {
-          loader.map('aurelia-framework', frameworkName);
-
-          return Promise.all([
-            loader.normalize('aurelia-dependency-injection', frameworkName)
-              .then(diName => loader.map('aurelia-dependency-injection', diName)),
-            loader.normalize('aurelia-router', bootstrapperName)
-              .then(routerName => loader.map('aurelia-router', routerName)),
-            loader.normalize('aurelia-logging-console', bootstrapperName)
-              .then(loggingConsoleName => loader.map('aurelia-logging-console', loggingConsoleName))
-          ]).then(() => {
-            return loader.loadModule(frameworkName).then(m => Aurelia = m.Aurelia);
-          });
-        });
-    });
-}
-
-function handleApp(loader, appHost) {
-  const moduleId = appHost.getAttribute('aurelia-app') || appHost.getAttribute('data-aurelia-app');
-  return config(loader, appHost, moduleId);
+      const frameworkPromise = map(PLATFORM.moduleName('aurelia-framework', { exports: ['Aurelia'] }), 
+                                   bootstrapperName);
+      // Please do NOT add PLATFORM.moduleName() around any of those modules.
+      // They are not actually loaded here, only mapped.
+      return Promise.all([
+        frameworkPromise,
+        frameworkPromise.then(frameworkName => map('aurelia-dependency-injection', frameworkName)),
+        map('aurelia-router', bootstrapperName),
+        map('aurelia-logging-console', bootstrapperName),
+      ]);
+    })
+    .then(([frameworkName,]) => loader.loadModule(frameworkName))
```

Comment:
Yes, I like it as it shows there is more in the array but I'm ignoring it.
I can remove it, the code would compile and work the same.

I also like trailing commas in arrays and objects (like line 121) as it's easier to add/remove items and it makes clean diffs (only the added/removed line is touched).